openroad_interface/graph_plotter.py

```diff
@@ -0,0 +1,44 @@
+import os
+
+import matplotlib.pyplot as plt
+import matplotlib.patches as mpatches
+import matplotlib
+import numpy as np
+import pandas as pd
+import seaborn as sns
+
+def box_whiskers_plot_design(type_data: str, designs: list, units: str, title: str, show_flier: bool) -> None:
+    '''
+    generates boxplots
+    '''
+    
+    box_plot = None
+    rcl = pd.read_csv('results/result_rcl.csv')
+    rcl[type_data] = np.log2(rcl[type_data])
+
+    sns.set_theme(rc={'figure.figsize':(10, 15)}, style="ticks")
+
+    flierprops = dict(marker='o', markerfacecolor='None', markersize=10,  markeredgecolor='black')
+    box_plot = sns.boxplot(data=rcl, x="design", y=type_data, hue="method", palette=["red", "blue"], width=0.5, showfliers=show_flier, flierprops=flierprops)
```

Comment:
pro tip.
Can just do 
plt.yscale('log') 
instead of manually converting the data to log.
and with this plt does some nice formatting stuff as well.